.github/workflows/build-test-deploy.yml

```diff
@@ -260,48 +262,85 @@ jobs:
       run: NO_APT_INSTALL=1 ./tests/test.sh --ignore=tests/test_executable.py
 
   build-container-aarch64:
-    if: startsWith(github.ref, 'refs/tags/')
-    needs: build-executable-aarch64
-    runs-on: ["self-hosted", "ARM64"]
+    runs-on:
+      - self-hosted
+      - public
+      - ARM64
+
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
```

Comment:
Keep this `fetch-depth`? No reason to remove.

Plus, you're missing one level of indentation that you've removed - please bring it back to match other lists in this YAML file (where the `-` marking each entry is indented by 2 spaces)

Reply:
933546bc4e20470e04d6bdf925072433b8cbe3d5